src/main/io/displayport_msp_osd.c

```diff
@@ -388,11 +429,35 @@ bool mspOsdSerialInit(void)
     return false;
 }
 
-displayPort_t* mspOsdDisplayPortInit(void)
+displayPort_t* mspOsdDisplayPortInit(const videoSystem_e videoSystem)
 {
     if (mspOsdSerialInit()) {
         init();
         displayInit(&mspOsdDisplayPort, &mspOsdVTable);
+        osdVideoSystem = videoSystem;
+        switch (videoSystem)
```

Comment:
Should I change this to an array of values and use videoSystem as the index (this would become 2 lines of code)